python/src/LinearModelValidation_doc.i.in

```diff
@@ -0,0 +1,140 @@
+key of the :class:`~openturns.ResourceMap` to `False`: use this
+option at your own risks.
+
+We suggest to use leave-one-out cross validation when possible,
+because it produces a more accurate estimate of the error than K-Fold does.
+If K-Fold is required, we suggest to use the largest possible value of :math:`k`.
+
+The predictions of the leave-one-one or K-Fold surrogate models
+are based on the hold-out method.
+For example, if we use the leave-one-out cross-validation method,
+the :math:`i`-th prediction is the prediction of the linear model
+trained using the hold-out sample where the :math:`i`-th observation
+was removed.
+This produces a sample of residuals which can be retrieved using
+the :class:`~openturns.LinearModelValidation.getResidualSample` method.
+The :class:`~openturns.LinearModelValidation.drawValidation` performs
+similarly.
+
+Examples
+--------
+Create a linear model.
+
+>>> import openturns as ot
+>>> func = ot.SymbolicFunction(
+...     ['x1', 'x2', 'x3'],
+...     ['x1 + x2 + sin(x2 * 2 * pi_) / 5 + 1e-3 * x3^2']
+... )
+>>> dimension = 3
+>>> distribution = ot.JointDistribution([ot.Normal()] * dimension)
+>>> sampleSize = 20
+>>> inputSample = distribution.getSample(sampleSize)
+>>> outputSample = func(inputSample)
+>>> algo = ot.LinearModelAlgorithm(inputSample, outputSample)
+>>> algo.run()
+>>> result = algo.getResult()
+
+Validate the linear model using leave-one-out cross-validation.
+
+>>> validation = ot.LinearModelValidation(result)
+
+We can use a specific cross-validation splitter if needed.
+
+>>> splitterLOO = ot.LeaveOneOutSplitter(sampleSize)
+>>> validation = ot.LinearModelValidation(result, splitterLOO)
+>>> r2Score = validation.computeR2Score()
+>>> print('R2 = ', r2Score[0])
+R2 =  0.98...
+
+Validate the linear model using K-Fold cross-validation.
+
+>>> splitterKFold = ot.KFoldSplitter(sampleSize)
+>>> validation = ot.LinearModelValidation(result, splitterKFold)
+
+Validate the linear model using K-Fold cross-validation and set K.
+
+>>> kFoldParameter = 10
+>>> splitterKFold = ot.KFoldSplitter(sampleSize, kFoldParameter)
+>>> validation = ot.LinearModelValidation(result, splitterKFold)
+
+Draw the validation graph.
+
+>>> graph = validation.drawValidation()"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelValidation::getLinearModelResult
+"Get the linear model result.
+
+Returns
+-------
+linearModelResult : :class:`~openturns.LinearModelResult`
+    The linear model result."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelValidation::getSplitter
+"Get the cross-validation method.
+
+Returns
+-------
+splitter : :class:`~openturns.SplitterImplementation`
+    The cross-validation method."
+
+// -------------------------------------v--------------------------------
+
+%feature("docstring") OT::LinearModelValidation::computePredictivityFactor
```

Comment:
Now that MetaModelValidation has a new name (see [here](https://openturns.github.io/openturns/master/user_manual/response_surface/_generated/openturns.MetaModelValidation.html?highlight=metamodelvalidation#openturns.MetaModelValidation.computeR2Score)), this should be named `computeR2Score`.